src/Maestro/SubscriptionActorService/PullRequestActor.cs

```diff
@@ -755,13 +755,17 @@ private async Task<string> CreatePullRequestAsync(List<UpdateAssetsParameters> u
 
             try
             {
-                string description = await CalculatePRDescriptionAndCommitUpdatesAsync(requiredUpdates, null, DarcRemoteFactory, targetRepository, newBranchName);
+                string description = await CalculatePRDescriptionAndCommitUpdatesAsync(
```

Comment:
In the case where the coherency update fails, but there are no non-coherency updates, I think this method will erroneously calculate an empty PR description. I think it should probably have a reasonable description

Reply:
You are right, the commit description will be empty in this case. However, the GitHub check will contain all the required information. Additionally, when the customer resolves the coherency issue, the `UpdatePullRequestAsync` will update the description. I may add a description like ~ "Coherency update failed. Please refer to the GitHub checks for more details." But it sounds awkward for me. @riarenas and/or @mmitche what do you think,

Reply:
I think we should title/describe the PR as if it was a coherency-only update like today, but note somewhere in the description that coherency has failed and the user should check the required checks,

If there are no commits because coherency failed, will AzDO and GitHub still open the PR? If not, maybe we add an empty commit with a commit message indicating that coherency failed.

Reply:
Covered this use case:
* If coherent only update failed - we add an empty commit and description. Example: https://github.com/maestro-auth-test/maestro-test3/pull/2122
* Added additional scenario test to cover this use case

Reply:
@mmitche could you please check the example and provide your feedback? Is the description and commit good enough?

Reply:
Going to update: "Please check Github checks for details." --> "Please review the GitHub checks for more information."